pkg/network/ebpf/c/protocols/protocol-dispatcher-helpers.h

```diff
@@ -0,0 +1,72 @@
+#ifndef __PROTOCOL_DISPATCHER_HELPERS_H
+#define __PROTOCOL_DISPATCHER_HELPERS_H
+
+#include <linux/types.h>
+
+#include "protocol-dispatcher-maps.h"
+#include "protocol-classification-helpers.h"
+#include "ip.h"
+
+// Returns true if the payload represents a TCP termination by checking if the tcp flags contains TCPHDR_FIN or TCPHDR_RST.
+static __always_inline bool is_tcp_termination(skb_info_t *skb_info) {
+    return skb_info->tcp_flags&(TCPHDR_FIN|TCPHDR_RST);
+}
+
+// A shared implementation for the runtime & prebuilt socket filter that classifies & dispatches the protocols of the connections.
+static __always_inline void protocol_dispatcher_entrypoint(struct __sk_buff *skb) {
+    skb_info_t skb_info = {0};
+    conn_tuple_t skb_tup = {0};
+
+    // Exporting the conn tuple from the skb, alongside couple of relevant fields from the skb.
+    if (!read_conn_tuple_skb(skb, &skb_info, &skb_tup)) {
+        return;
+    }
+
+    // We handle the payload if it is an non empty TCP packet, or termination TCP packet.
```

Comment:
same here, rephrase the comment to support the code.. (btw currently the comment doesn't match the condition)

e.g: "we don't process non tcp packets, nor empty tcp packets which are not tcp termination packets"